interfaces/cython/cantera/yaml2ck.py

```diff
@@ -0,0 +1,1047 @@
+            name_and_note = f"{species.name} {note[species.name]}"
+            text = [comment[species.name]]
+            text.append(
+                f"{name_and_note:<24}"  # name and date/note field
+                + f"{composition_string:<20}"
+                + "G"  # only supports gas phase
+                + f"{species.thermo.min_temp:10.3f}"
+                + f"{species.thermo.max_temp:10.3f}"
+                + f"{species.thermo.coeffs[0]:10.3f}"
+                + f"{1:>5}"
+            )
+
+            # second line has first five coefficients of high-temperature range,
+            # ending with a "2" in column 79
+            text.append(
+                "".join([f"{c:15.8e}" for c in species.thermo.coeffs[1:6]]) + f"{2:>5}"
+            )
+
+            # third line has the last two coefficients of the high-temperature range,
+            # first three coefficients of low-temperature range, and "3"
+            text.append(
+                "".join([f"{c:15.8e}" for c in species.thermo.coeffs[6:8]])
+                + "".join([f"{c:15.8e}" for c in species.thermo.coeffs[8:11]])
+                + f"{3:>5}"
+            )
+
+            # fourth and last line has the last four coefficients of the
+            # low-temperature range, and "4"
+            text.append(
+                "".join([f"{c:15.8e}" for c in species.thermo.coeffs[11:15]])
+                + f"{4:>20}\n"
+            )
+
+            thermo_text.append("\n".join(text))
+
+        return thermo_text
+
+    def build_nasa9_text(solution_species):  # TODO: Nonfunctional, general scheme
+        """
+        Would build NASA9-style text string however, Chemkin doesn't accept NASA9
+        currently
+        """
+        # Ex:
+        #   therm NASA9
+        #     200.00  1000.00  6000.00  20000.   3/19/02
+        # e-                Ref-Species. Chase, 1998 3/82.
+        #  3 912/98 E   1.00    0.00    0.00    0.00    0.00 0.000548579903          0.000
+        #     298.150   1000.0007 -2.0 -1.0  0.0  1.0  2.0  3.0  4.0  0.0         6197.428
+        #  0.000000000D+00 0.000000000D+00 2.500000000D+00 0.000000000D+00 0.000000000D+00
+        #  0.000000000D+00 0.000000000D+00                -7.453750000D+02-1.172081224D+01
+        #    1000.000   6000.0007 -2.0 -1.0  0.0  1.0  2.0  3.0  4.0  0.0         6197.428
+        #  0.000000000D+00 0.000000000D+00 2.500000000D+00 0.000000000D+00 0.000000000D+00
+        #  0.000000000D+00 0.000000000D+00                -7.453750000D+02-1.172081224D+01
+        #    6000.000  20000.0007 -2.0 -1.0  0.0  1.0  2.0  3.0  4.0  0.0         6197.428
+        #  0.000000000D+00 0.000000000D+00 2.500000000D+00 0.000000000D+00 0.000000000D+00
+        #  0.000000000D+00 0.000000000D+00                -7.453750000D+02-1.172081224D+01
+        #
+        # Could build NASA9 from NASA7 by setting first 2 coefficients to 0
+        # NASA9 not supported by Chemkin currently though? Comes from NASA CEA mostly
+        pass
```

Comment:
As this is essentially commented out code, I'd suggest deleting it.

Reply:
That's fine. I had left it in there for myself for if Chemkin decided to implement NASA9